types/flatbuffers/index.d.ts

```diff
@@ -167,7 +169,8 @@ declare global {
        */
       endObject(): Offset;
 
-      finish(root_table: Offset, file_identifier?: string): void;
+      finish(root_table: Offset, file_identifier?: string, size_prefix?: boolean): void;
```

Comment:
as you ticket the test options, can we have the test for the method udpated, e.g. like:
```ts
  static finishMonsterBuffer(builder: flatbuffers.Builder, offset: flatbuffers.Offset) {
    builder.finish(offset);
    builder.finish(offset, 'MONS');
    builder.finish(offset, 'MONS', true);
  }
```